static/main.js

```diff
@@ -57,28 +57,34 @@
 (function() {
   var games = document.getElementsByTagName('dd');
   var tags = document.getElementsByClassName('tag');
+  var activeTag;
 
   for (var i = 0, l = tags.length; i < l; i += 1) {
     tags[i].addEventListener('click', onclick);
   }
 
   function onclick(e) {
     var t = e.target.hasAttribute('data-name') ? e.target : e.target.parentNode;
+    var curTag = t.getAttribute('data-name');
     var parent;
     var game, gameTags;
 
-    if (!hasClass(document.body, 'tags-active')) {
-      document.body.className += ' tags-active';
+    if (curTag === activeTag) {
+      activeTag = false;
```

Comment:
I would prefer seeing `null` here. It's not a big deal, but it's semantically more correct (the same with `highlightTags` argument lower - `false` vs `null`).